cg/apps/hermes/models.py

```diff
@@ -16,23 +17,24 @@ class CGTag(BaseModel):
     mandatory: Optional[bool] = False
 
 
+def check_mandatory_exists(file: CGTag):
+    if file.mandatory:
+        assert Path(file.path).exists(), f"Mandatory file cannot be found at {file.path}"
```

Comment:
I would like to avoid using "assert" in production code.

Reply:
Seems to be the way validators are written - should we avoid them then? See e.g. https://docs.pydantic.dev/latest/usage/validators/

Reply:
From Pydantic docs: Warning: If you make use of assert statements, keep in mind that running Python with the -O optimization flag disables assert statements, and validators will stop working.

I do not think it is worth the gain when it so easily replaced.

Reply:
Exchanged for a "raise error"